src/services/classifier2020.ts

```diff
@@ -0,0 +1,257 @@
+/** @internal */
+namespace ts.classifier.modern {
+
+    /** @internal */
+    export enum TokenEncodingConsts {
+        typeOffset = 8,
+        modifierMask = (1 << typeOffset) - 1
+    }
+
+    /** @internal */
+    export enum TokenType {
+        class, enum, interface, namespace, typeParameter, type, parameter, variable, enumMember, property, function, member, _

```

Comment:
What do these `_` members represent?

Reply:
I think the original source probably used them to figure out how long the enum was, but it's not being used anymore so I've dropped them 👍